src/App.js

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import './App.scss';
```

Comment:
Nice job on this task, it would, however, be neater if we could l reduce importing every single .scss file to each component. For instance, it would be more useful to have the bootstrap styles imported in the single topmost styles file which could be `index.scss` or the `app.scss`. Kindly consider streamlining the way we import and reuse styles.